### DISCUSSION TITLE
Machine does not connect when power supply is connected

### DISCUSSION THREAD

**Discussion by jagjit29** [arkypita/LaserGRBL#1921] [General]
New downloaded software have all functions visible but it is not able to connect to all 3 machines two tree machines i bought recently,if i plug just usb cable it works ok ,but as soon as i give 12 volt external supply it stops working. if i  give supply initially along with connected usb cable it doesnot connect at all.i have checked all normal trouble shoot points device manager shows driver ch340, same port i use on grbl software. 4 days i am trying but all in vain, even company also not able to sort out, please help

**Comment by arkypita** [comment]
Not jet solved?

If you want to help me understand why it does not connect, please send me a copy of your session log (menu ?, open session log) and perform a full com-log (communication log) between lasergrbl and the machine.

This can be done starting LaserGRBL, click on menu ?, activate extended log, and then connecting to the machine.
After 10 seconds, when the connect fail, stop extended log (again from menu ?) and copy results here.

**Reply by jagjit29** [reply]
yes please check file
00000	00357684	log	Recording session started @ 12-09-2022 17:11:35
00001	00391398	com	Open COM12 @ 115200 baud Ctrl-X (UsbSerial)
00002	00397078	com	Open COM1 @ 115200 baud Ctrl-X (UsbSerial)
00003	00408914	com	Open COM16 @ 115200 baud Ctrl-X (UsbSerial)
00004	00409106	tx	[18]
00005	00409152	tx	[3F]
00006	00409404	tx	[3F]
00007	00409654	tx	[3F]
00008	00409904	tx	[3F]
00009	00410154	tx	[3F]
00010	00410404	tx	[3F]
00011	00410653	tx	[3F]
00012	00410903	tx	[3F]
00013	00411153	tx	[3F]
00014	00411403	tx	[3F]
00015	00411653	tx	[3F]
00016	00411903	tx	[3F]
00017	00412154	tx	[3F]
00018	00412405	tx	[3F]
00019	00412653	tx	[3F]
00020	00412904	tx	[3F]
00021	00413154	tx	[3F]
00022	00413404	tx	[3F]
00023	00413653	tx	[3F]
00024	00413903	tx	[3F]
00025	00414160	tx	[3F]
00026	00414403	tx	[3F]
00027	00414653	tx	[3F]
00028	00414903	tx	[3F]
00029	00415153	tx	[3F]
00030	00415403	tx	[3F]
00031	00415653	tx	[3F]
00032	00415904	tx	[3F]
00033	00416154	tx	[3F]
00034	00416404	tx	[3F]
00035	00416654	tx	[3F]
00036	00416904	tx	[3F]
00037	00417153	tx	[3F]
00038	00417403	tx	[3F]
00039	00417653	tx	[3F]
00040	00417903	tx	[3F]
00041	00418154	tx	[3F]
00042	00418405	tx	[3F]
00043	00418654	tx	[3F]
00044	00418903	tx	[3F]
00045	00418919	com	Close COM16 [CORE]
00046	00429406	log	Recording session stopped @ 12-09-2022 17:12:46

**Reply by jagjit29** [reply]
now if we don't connect 12 v power supply to board  and just connect usb cable to mother board. results are like this
00000	00740737	log	Recording session started @ 12-09-2022 17:17:58
00001	00747234	com	Open COM16 @ 115200 baud Ctrl-X (UsbSerial)
00002	00747237	tx	[18]
00003	00747238	tx	[3F]
00004	00747266	rx	\r
00005	00747266	rx	Grbl 1.1f ['$' for help]\r
00006	00747274	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0|WCO:0.000,0.000,0.000>\r
00007	00747490	tx	[3F]
00008	00747496	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0|Ov:100,100,100|A:F>\r
00009	00747740	tx	[3F]
00010	00747744	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00011	00747990	tx	[3F]
00012	00747994	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00013	00748255	tx	[3F]
00014	00748261	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00015	00748309	tx	$$\n
00016	00748313	rx	$0=10\r
00017	00748318	rx	$1=25\r
00018	00748318	rx	$2=0\r
00019	00748318	rx	$3=2\r
00020	00748318	rx	$4=0\r
00021	00748318	rx	$5=0\r
00022	00748319	rx	$6=0\r
00023	00748319	rx	$10=1\r
00024	00748319	rx	$11=0.010\r
00025	00748319	rx	$12=0.002\r
00026	00748319	rx	$13=0\r
00027	00748319	rx	$20=0\r
00028	00748319	rx	$21=0\r
00029	00748321	rx	$22=0\r
00030	00748321	rx	$23=1\r
00031	00748321	rx	$24=100.000\r
00032	00748324	rx	$25=1000.000\r
00033	00748324	rx	$26=250\r
00034	00748324	rx	$27=1.000\r
00035	00748327	rx	$30=1000\r
00036	00748327	rx	$31=0\r
00037	00748327	rx	$32=1\r
00038	00748327	rx	$100=80.000\r
00039	00748329	rx	$101=80.000\r
00040	00748329	rx	$102=800.000\r
00041	00748332	rx	$110=6000.000\r
00042	00748332	rx	$111=6000.000\r
00043	00748335	rx	$112=600.000\r
00044	00748335	rx	$120=500.000\r
00045	00748338	rx	$121=500.000\r
00046	00748338	rx	$122=10.000\r
00047	00748338	rx	$130=300.000\r
00048	00748340	rx	$131=300.000\r
00049	00748340	rx	$132=50.000\r
00050	00748340	rx	ok\r
00051	00748489	tx	[3F]
00052	00748494	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00053	00748739	tx	[3F]
00054	00748743	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00055	00748989	tx	[3F]
00056	00748994	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00057	00749240	tx	[3F]
00058	00749244	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00059	00749490	tx	[3F]
00060	00749494	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00061	00749740	tx	[3F]
00062	00749746	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0|WCO:0.000,0.000,0.000>\r
00063	00749990	tx	[3F]
00064	00749996	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0|Ov:100,100,100|A:F>\r
00065	00750239	tx	[3F]
00066	00750244	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00067	00750489	tx	[3F]
00068	00750493	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00069	00750739	tx	[3F]
00070	00750743	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00071	00750990	tx	[3F]
00072	00750994	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00073	00751240	tx	[3F]
00074	00751244	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00075	00751490	tx	[3F]
00076	00751494	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00077	00751740	tx	[3F]
00078	00751744	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00079	00751989	tx	[3F]
00080	00751993	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00081	00752239	tx	[3F]
00082	00752245	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0|WCO:0.000,0.000,0.000>\r
00083	00752489	tx	[3F]
00084	00752495	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0|Ov:100,100,100|A:F>\r
00085	00752740	tx	[3F]
00086	00752744	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00087	00752990	tx	[3F]
00088	00752994	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00089	00753240	tx	[3F]
00090	00753244	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00091	00753490	tx	[3F]
00092	00753494	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00093	00753739	tx	[3F]
00094	00753744	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00095	00753989	tx	[3F]
00096	00753993	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00097	00754240	tx	[3F]
00098	00754244	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00099	00754490	tx	[3F]
00100	00754494	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00101	00754740	tx	[3F]
00102	00754746	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0|WCO:0.000,0.000,0.000>\r
00103	00754990	tx	[3F]
00104	00754996	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0|Ov:100,100,100|A:F>\r
00105	00755239	tx	[3F]
00106	00755244	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00107	00755489	tx	[3F]
00108	00755494	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00109	00755739	tx	[3F]
00110	00755743	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00111	00755990	tx	[3F]
00112	00755994	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00113	00756240	tx	[3F]
00114	00756244	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00115	00756490	tx	[3F]
00116	00756494	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00117	00756740	tx	[3F]
00118	00756745	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00119	00756989	tx	[3F]
00120	00756994	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00121	00757239	tx	[3F]
00122	00757246	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0|WCO:0.000,0.000,0.000>\r
00123	00757489	tx	[3F]
00124	00757495	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0|Ov:100,100,100|A:F>\r
00125	00757740	tx	[3F]
00126	00757744	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00127	00757990	tx	[3F]
00128	00757994	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00129	00758240	tx	[3F]
00130	00758244	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00131	00758490	tx	[3F]
00132	00758493	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00133	00758739	tx	[3F]
00134	00758744	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00135	00758989	tx	[3F]
00136	00758993	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00137	00759240	tx	[3F]
00138	00759244	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00139	00759490	tx	[3F]
00140	00759494	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00141	00759740	tx	[3F]
00142	00759746	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0|WCO:0.000,0.000,0.000>\r
00143	00759990	tx	[3F]
00144	00759995	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0|Ov:100,100,100|A:F>\r
00145	00760239	tx	[3F]
00146	00760244	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00147	00760489	tx	[3F]
00148	00760493	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00149	00760739	tx	[3F]
00150	00760744	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00151	00760989	tx	[3F]
00152	00760993	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00153	00761002	log	Recording session stopped @ 12-09-2022 17:18:18

**Comment by arkypita** [comment]
Given this log seem that the machine give no reply to LaserGRBL status request.
Are you sure that this machine use grbl firmware?

This is not a LaserGRBL issue, I suggest you to contact Two Trees customer support.

**Comment by arkypita** [comment]
I see... when you don't connect 12v power supply the responses are correct, but when connected no response!
I have no idea of why the machine should behave like this. Which model is it?

**Reply by jagjit29** [reply]
![IMG_20220906_130844](https://user-images.githubusercontent.com/112888495/189884941-42a1905b-d63a-41c1-86bc-e1a44ec01dab.jpg)
See the motherboard is this

**Reply by arkypita** [reply]
I asked for machine model, do you have any link to product?

**Reply by jagjit29** [reply]
TT-5.5

**Reply by arkypita** [reply]
From what I see the machine should be perfectly compatible and there is no reason for it to behave like this.

I can hypothesize a hardware defect, for example a short circuit on the power supply tracks, or a defect in the power supply you are using (maybe shorted) or some chip burnt.

**Reply by arkypita** [reply]
You have 3 machine and all 3 does the same?
Did you try with a different PC?
Did you try with a laptop disconnected from power supply?
Did you try with different power supply? Different USB cable?

**Reply by jagjit29** [reply]
yes same for all machines
yes i tried different pc
yes different power supply and usb cable tried
yes i do it mostly with labtop
but i tried with PC too

**Reply by arkypita** [reply]
Did they never work or did all three of them stop working at some point?
- If they never worked, I'm guessing a bad batch of controller board.
- If all three of them stop working at some point, i guess a destructive event such as lightning on the power line.

You've also tried other software like lightburn, SourceRabbit or [Universal G-code Sender (UGS)](https://github.com/winder/Universal-G-Code-Sender)? This is to rule out a LaserGRBL problem (but I'm 99.999999% sure looking at the log that LaserGRBL has nothing to do with your issue).

I literally have no other ideas, i think you need to continue this communication with the manufacturer or seller of these machines.

**Reply by arkypita** [reply]
From the photo you sent me it seems that you have also tested the control board with the motors and the laser disconnected (is it correct?) so you can exclude a short of the motors, the laser and the cables...

It's really weird, but I don't know how to help you. Any other particular effect that you notice when you connect the 12V, such as LEDs that go out or similar?

Can you do this test? Connect LaserGRBL with the 12V disconnected and connect the 12V while LaserGRBL is connected to the board and is talking to it correctly. I want to see if the controller send any strange data when you connect the power supply.
Obviously by registering the comlog during the test. (open LaserGRBL, start com logging, connect the control board **without 12V plugged**, only at this point plug 12V.)

**Reply by jagjit29** [reply]
1. Light burn also not working i have checked
2. now i want to use mks cards off line system please suggest which model and TFT is better
3. TWO TREE technicians are clueless too about this problem, now they promised t send new electronic cards ,but i want to try MKS cards also

**Reply by jagjit29** [reply]
what you suggest i had done after connection when 12 v connected it stops i will send you log data soon

**Reply by jagjit29** [reply]
this log session, first usb connected then software connected then 12 v plugged later it stops responding
00000	01376475	log	Recording session started @ 14-09-2022 16:26:15
00001	01383087	com	Open COM15 @ 115200 baud Ctrl-X (UsbSerial)
00002	01383142	tx	[18]
00003	01383171	tx	[3F]
00004	01383422	tx	[3F]
00005	01383672	tx	[3F]
00006	01383755	rx	error:7\r
00007	01383768	rx	$0=10\r
00008	01383780	rx	$1=25\r
00009	01383782	rx	$2=0\r
00010	01383783	rx	$3=2\r
00011	01383783	rx	$4=0\r
00012	01383783	rx	$5=0\r
00013	01383783	rx	$6=0\r
00014	01383783	rx	$10=1\r
00015	01383783	rx	$11=0.010\r
00016	01383783	rx	$12=0.002\r
00017	01383784	rx	$13=0\r
00018	01383784	rx	$20=0\r
00019	01383784	rx	$21=0\r
00020	01383784	rx	$22=0\r
00021	01383784	rx	$23=1\r
00022	01383785	rx	$24=100.000\r
00023	01383785	rx	$25=1000.000\r
00024	01383785	rx	$26=250\r
00025	01383785	rx	$27=1.000\r
00026	01383785	rx	$30=1000\r
00027	01383786	rx	$31=0\r
00028	01383786	rx	$32=1\r
00029	01383786	rx	$100=80.000\r
00030	01383786	rx	$101=80.000\r
00031	01383786	rx	$102=800.000\r
00032	01383786	rx	$110=6000.000\r
00033	01383786	rx	$111=6000.000\r
00034	01383786	rx	$112=600.000\r
00035	01383787	rx	$120=500.000\r
00036	01383787	rx	$121=500.000\r
00037	01383787	rx	$122=10.000\r
00038	01383787	rx	$130=300.000\r
00039	01383787	rx	$131=300.000\r
00040	01383787	rx	$132=50.000\r
00041	01383803	rx	\r
00042	01383804	rx	Grbl 1.1f ['$' for help]\r
00043	01383922	tx	[3F]
00044	01383928	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0|WCO:0.000,0.000,0.000>\r
00045	01384173	tx	[3F]
00046	01384178	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0|Ov:100,100,100|A:F>\r
00047	01384423	tx	[3F]
00048	01384427	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00049	01384509	tx	$$\n
00050	01384513	rx	$0=10\r
00051	01384513	rx	$1=25\r
00052	01384513	rx	$2=0\r
00053	01384513	rx	$3=2\r
00054	01384513	rx	$4=0\r
00055	01384515	rx	$5=0\r
00056	01384515	rx	$6=0\r
00057	01384516	rx	$10=1\r
00058	01384516	rx	$11=0.010\r
00059	01384518	rx	$12=0.002\r
00060	01384518	rx	$13=0\r
00061	01384518	rx	$20=0\r
00062	01384518	rx	$21=0\r
00063	01384521	rx	$22=0\r
00064	01384521	rx	$23=1\r
00065	01384521	rx	$24=100.000\r
00066	01384523	rx	$25=1000.000\r
00067	01384524	rx	$26=250\r
00068	01384524	rx	$27=1.000\r
00069	01384526	rx	$30=1000\r
00070	01384526	rx	$31=0\r
00071	01384526	rx	$32=1\r
00072	01384526	rx	$100=80.000\r
00073	01384529	rx	$101=80.000\r
00074	01384529	rx	$102=800.000\r
00075	01384532	rx	$110=6000.000\r
00076	01384532	rx	$111=6000.000\r
00077	01384534	rx	$112=600.000\r
00078	01384534	rx	$120=500.000\r
00079	01384537	rx	$121=500.000\r
00080	01384537	rx	$122=10.000\r
00081	01384537	rx	$130=300.000\r
00082	01384540	rx	$131=300.000\r
00083	01384540	rx	$132=50.000\r
00084	01384540	rx	ok\r
00085	01384673	tx	[3F]
00086	01384677	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00087	01384923	tx	[3F]
00088	01384927	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00089	01385173	tx	[3F]
00090	01385176	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00091	01385423	tx	[3F]
00092	01385427	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00093	01385672	tx	[3F]
00094	01385676	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00095	01385922	tx	[3F]
00096	01385927	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00097	01386175	tx	[3F]
00098	01386179	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00099	01386422	tx	[3F]
00100	01386429	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0|WCO:0.000,0.000,0.000>\r
00101	01386672	tx	[3F]
00102	01386678	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0|Ov:100,100,100|A:F>\r
00103	01386922	tx	[3F]
00104	01386926	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00105	01387174	tx	[3F]
00106	01387179	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00107	01387422	tx	[3F]
00108	01387426	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00109	01387676	tx	[3F]
00110	01387680	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00111	01387923	tx	[3F]
00112	01387927	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00113	01388173	tx	[3F]
00114	01388190	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00115	01388423	tx	[3F]
00116	01388427	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00117	01388673	tx	[3F]
00118	01388677	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0>\r
00119	01388923	tx	[3F]
00120	01388929	rx	<Idle|MPos:0.000,0.000,0.000|FS:0,0|WCO:0.000,0.000,0.000>\r
00121	01388968	tx	$J=G91X10.0F1000\n
00122	01388973	rx	ok\r
00123	01389180	tx	[3F]
00124	01389186	rx	<Jog|MPos:3.200,0.000,0.000|FS:1000,0|Ov:100,100,100|A:F>\r
00125	01389422	tx	[3F]
00126	01389427	rx	<Jog|MPos:7.238,0.000,0.000|FS:1000,0>\r
00127	01389672	tx	[3F]
00128	01389677	rx	<Idle|MPos:10.000,0.000,0.000|FS:0,0>\r
00129	01389922	tx	[3F]
00130	01389927	rx	<Idle|MPos:10.000,0.000,0.000|FS:0,0>\r
00131	01390172	tx	[3F]
00132	01390177	rx	<Idle|MPos:10.000,0.000,0.000|FS:0,0>\r
00133	01390422	tx	[3F]
00134	01390427	rx	<Idle|MPos:10.000,0.000,0.000|FS:0,0>\r
00135	01390672	tx	[3F]
00136	01390676	rx	<Idle|MPos:10.000,0.000,0.000|FS:0,0>\r
00137	01390922	tx	[3F]
00138	01390926	rx	<Idle|MPos:10.000,0.000,0.000|FS:0,0>\r
00139	01391172	tx	[3F]
00140	01391176	rx	<Idle|MPos:10.000,0.000,0.000|FS:0,0>\r
00141	01391423	tx	[3F]
00142	01391429	rx	<Idle|MPos:10.000,0.000,0.000|FS:0,0|WCO:0.000,0.000,0.000>\r
00143	01391673	tx	[3F]
00144	01391677	rx	<Idle|MPos:10.000,0.000,0.000|FS:0,0>\r
00145	01391923	tx	[3F]
00146	01391927	rx	<Idle|MPos:10.000,0.000,0.000|FS:0,0>\r
00147	01392173	tx	[3F]
00148	01392177	rx	<Idle|MPos:10.000,0.000,0.000|FS:0,0>\r
00149	01392423	tx	[3F]
00150	01392427	rx	<Idle|MPos:10.000,0.000,0.000|FS:0,0>\r
00151	01392672	tx	[3F]
00152	01392923	tx	[3F]
00153	01393172	tx	[3F]
00154	01393422	tx	[3F]
00155	01393672	tx	[3F]
00156	01393922	tx	[3F]
00157	01394172	tx	[3F]
00158	01394422	tx	[3F]
00159	01394672	tx	[3F]
00160	01394922	tx	[3F]
00161	01395003	tx	$J=G91X10.0F1000\n
00162	01395173	tx	[3F]
00163	01395423	tx	[3F]
00164	01395673	tx	[3F]
00165	01395923	tx	[3F]
00166	01395973	tx	$J=G91X10.0F1000\n
00167	01396173	tx	[3F]
00168	01396423	tx	[3F]
00169	01396672	tx	[3F]
00170	01396922	tx	[3F]
00171	01397172	tx	[3F]
00172	01397422	tx	[3F]
00173	01397672	tx	[3F]
00174	01397922	tx	[3F]
00175	01398172	tx	[3F]
00176	01398423	tx	[3F]
00177	01398673	tx	[3F]
00178	01398923	tx	[3F]
00179	01399173	tx	[3F]
00180	01399423	tx	[3F]
00181	01399673	tx	[3F]
00182	01399922	tx	[3F]
00183	01400172	tx	[3F]
00184	01400422	tx	[3F]
00185	01400672	tx	[3F]
00186	01400716	log	Recording session stopped @ 14-09-2022 16:26:39

**Reply by StuartB4** [reply]
Are you sure your 12volt supply is actually 12volts. You may need to check it with a multimeter.

**Comment by jagjit29** [comment]
Yes ,checked,i changed power supply also

**Comment by arkypita** [comment]
Any update?

**Comment by jagjit29** [comment]
Changing motherboard worked

**Comment by jagjit29** [comment]
Hello
Happy New Year's
I am using  CNC software UGS..I m facing problem in importing heavy  SVG
files.Please help n suggest what is issue
